### ISSUE TITLE
Option to disable team fail in Team VS

### ISSUE THREAD

**Issue by nautzz** [ppy/osu-stable-issues#209]
Many tourneys nowadays count failed scores, which is easily handled in 1v1 tourneys (just use free for all), but in team tourneys there's the issue of team fails. Since this can be worked around by forcing everyone to use nofail, it's definitely low priority, but if it's easy to implement it would be greatly appreciated (couldn't it function the same way as in FFA, where it just allows everyone to continue playing?)

Something like "!mp teamfail on/off" would be preferable.

**Comment by smoogipoo** [comment]
This should be a player decision, not a tournament host decision.

**Comment by nautzz** [comment]
Feel like there was a misunderstanding here; I don't want an option to force nofail, I simply want an option to disable the game ending when all members of a team have failed (much like the game continues in FFA even if all players have failed). 

Team fail renders the point of having a rule allowing failed scores moot, since the game just ends.

**Comment by smoogipoo** [comment]
Is there a reason why the players can't put NF on themselves?

**Comment by nautzz** [comment]
When playing maps in pools that aren't freemod, would be a good example. Could just always use freemod regardless of what pool the map is from, I suppose.

It also messes a bit with the display of scores on stream in the tourney client, if one team is using NF and the other isn't.

**Comment by smoogipoo** [comment]
If you're playing non-freemod maps, why not use `!mp mods HR NF` ? Maybe I'm misunderstanding something.

**Comment by nautzz** [comment]
That's true I suppose. As I said in the initial post, it is ultimately low priority, there are workarounds, but it feels a bit hacky and strange that it's required as soon as you want to organize a tourney that isn't 1v1.

All of the reasons for having this feature added feel likely to be considered "low priority" (for perfectly understandable reasons), like another one that comes to mind is how it affects the collection of stats from matches. Qualifiers are typically FFA regardless, meaning there's no need for forcing NF, whereas that isn't the case for bracket stage. Realistically this doesn't have much of an effect on the tourney, since the data in bracket stage won't have an effect on seeding or anything like that in almost all formats.

Basically, there are a bunch of annoyances I associate with this all, but ultimately it's low priority. I suppose it comes down to how hard it is to implement, and the cost-benefit analysis on your end.

**Comment by nautzz** [comment]
There is also this bug: https://osu.ppy.sh/community/forums/topics/942676

This happened in the tourney I'm currently organizing in a match today.

**Comment by smoogipoo** [comment]
The linked issue has been addressed, and will likely be in the next cutting edge build.
